hd/README.md

```diff
@@ -0,0 +1,44 @@
+## Introduction
+
+This program adds `# ` to the beginning of the stdin and prints it.
```

Comment:
![image](https://github.com/sdbx/Utils/assets/83723320/794ffb43-519e-41fc-8a5c-b2dac4131665)
Sounds awesome! It must be useful to do some buntangs!